dashboard/src/components/InstitutionLogo.vue

```diff
@@ -0,0 +1,28 @@
+<script setup lang="ts">
+import { defineProps } from "vue";
+
+const props = defineProps<{
+  logo: string;
+  name: string;
+  url: string;
+}>();
+
+const imgTag = `<img
+    id="institution-logo"
+    src="${props.logo}"
+    alt="${props.name}"
+    height="30"
+    class="d-none d-sm-block mx-3"
```

Comment:
This classes should be added to the link or div containers, otherwise we "show" the link without content. Also, why a constant with v-html instead of doing it in the template directly?

Reply:
I used a variable and v-html to avoid having to repeat the HTML for the logo twice, like so:

```
<a
    v-if="props.logo && props.url"
    :href="props.url"
    target="_blank"
    rel="external"
  >
    <img
      id="institution-logo"
      src="${props.logo}"
      alt="${props.name}"
      height="30"
      class="d-none d-sm-block mx-3"
    />
  </a>
  <div v-else-if="props.logo">
    <img
      id="institution-logo"
      src="${props.logo}"
      alt="${props.name}"
      height="30"
      class="d-none d-sm-block mx-3"
    />
  </div>
```

I haven't figured out a really good way to  with Vue's templating system to conditionally add a link that doesn't require repeating any nested elements. :(